INFO - 2024-06-16 14:58:31 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-06-16 14:58:33 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
ERROR - 2024-06-16 14:58:58 --> Error connecting to the database: mysqli_sql_exception: No connection could be made because the target machine actively refused it in C:\xampp2\htdocs\codeigniter\codeigniter4\codeigniter4-framework-9b2cd73\system\Database\MySQLi\Connection.php:186
Stack trace:
#0 C:\xampp2\htdocs\codeigniter\codeigniter4\codeigniter4-framework-9b2cd73\system\Database\MySQLi\Connection.php(186): mysqli->real_connect('localhost', 'root', Object(SensitiveParameterValue), 'test', 3306, '', 0)
#1 C:\xampp2\htdocs\codeigniter\codeigniter4\codeigniter4-framework-9b2cd73\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp2\htdocs\codeigniter\codeigniter4\codeigniter4-framework-9b2cd73\system\Database\BaseConnection.php(604): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp2\htdocs\codeigniter\codeigniter4\codeigniter4-framework-9b2cd73\system\Database\BaseBuilder.php(1629): CodeIgniter\Database\BaseConnection->query('SELECT 1\nFROM `...', Array, false)
#4 C:\xampp2\htdocs\codeigniter\codeigniter4\codeigniter4-framework-9b2cd73\system\Validation\StrictRules\Rules.php(228): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp2\htdocs\codeigniter\codeigniter4\codeigniter4-framework-9b2cd73\system\Validation\Validation.php(336): CodeIgniter\Validation\StrictRules\Rules->is_unique('vanessa.cyrilla...', 'email', Array, NULL, 'email')
#6 C:\xampp2\htdocs\codeigniter\codeigniter4\codeigniter4-framework-9b2cd73\system\Validation\Validation.php(201): CodeIgniter\Validation\Validation->processRules('email', 'email', 'vanessa.cyrilla...', Array, Array, 'email')
#7 C:\xampp2\htdocs\codeigniter\codeigniter4\codeigniter4-framework-9b2cd73\system\Controller.php(137): CodeIgniter\Validation\Validation->run()
#8 C:\xampp2\htdocs\codeigniter\codeigniter4\codeigniter4-framework-9b2cd73\app\Controllers\Auth.php(72): CodeIgniter\Controller->validate(Array)
#9 C:\xampp2\htdocs\codeigniter\codeigniter4\codeigniter4-framework-9b2cd73\system\CodeIgniter.php(933): App\Controllers\Auth->save()
#10 C:\xampp2\htdocs\codeigniter\codeigniter4\codeigniter4-framework-9b2cd73\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
#11 C:\xampp2\htdocs\codeigniter\codeigniter4\codeigniter4-framework-9b2cd73\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 C:\xampp2\htdocs\codeigniter\codeigniter4\codeigniter4-framework-9b2cd73\system\Boot.php(312): CodeIgniter\CodeIgniter->run()
#13 C:\xampp2\htdocs\codeigniter\codeigniter4\codeigniter4-framework-9b2cd73\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#14 C:\xampp2\htdocs\codeigniter\codeigniter4\codeigniter4-framework-9b2cd73\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#15 C:\xampp2\htdocs\codeigniter\codeigniter4\codeigniter4-framework-9b2cd73\system\Commands\Server\rewrite.php(49): require_once('C:\\xampp2\\htdoc...')
#16 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: No connection could be made because the target machine actively refused it in C:\xampp2\htdocs\codeigniter\codeigniter4\codeigniter4-framework-9b2cd73\system\Database\MySQLi\Connection.php:231
Stack trace:
#0 C:\xampp2\htdocs\codeigniter\codeigniter4\codeigniter4-framework-9b2cd73\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp2\htdocs\codeigniter\codeigniter4\codeigniter4-framework-9b2cd73\system\Database\BaseConnection.php(604): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp2\htdocs\codeigniter\codeigniter4\codeigniter4-framework-9b2cd73\system\Database\BaseBuilder.php(1629): CodeIgniter\Database\BaseConnection->query('SELECT 1\nFROM `...', Array, false)
#3 C:\xampp2\htdocs\codeigniter\codeigniter4\codeigniter4-framework-9b2cd73\system\Validation\StrictRules\Rules.php(228): CodeIgniter\Database\BaseBuilder->get()
#4 C:\xampp2\htdocs\codeigniter\codeigniter4\codeigniter4-framework-9b2cd73\system\Validation\Validation.php(336): CodeIgniter\Validation\StrictRules\Rules->is_unique('vanessa.cyrilla...', 'email', Array, NULL, 'email')
#5 C:\xampp2\htdocs\codeigniter\codeigniter4\codeigniter4-framework-9b2cd73\system\Validation\Validation.php(201): CodeIgniter\Validation\Validation->processRules('email', 'email', 'vanessa.cyrilla...', Array, Array, 'email')
#6 C:\xampp2\htdocs\codeigniter\codeigniter4\codeigniter4-framework-9b2cd73\system\Controller.php(137): CodeIgniter\Validation\Validation->run()
#7 C:\xampp2\htdocs\codeigniter\codeigniter4\codeigniter4-framework-9b2cd73\app\Controllers\Auth.php(72): CodeIgniter\Controller->validate(Array)
#8 C:\xampp2\htdocs\codeigniter\codeigniter4\codeigniter4-framework-9b2cd73\system\CodeIgniter.php(933): App\Controllers\Auth->save()
#9 C:\xampp2\htdocs\codeigniter\codeigniter4\codeigniter4-framework-9b2cd73\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
#10 C:\xampp2\htdocs\codeigniter\codeigniter4\codeigniter4-framework-9b2cd73\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp2\htdocs\codeigniter\codeigniter4\codeigniter4-framework-9b2cd73\system\Boot.php(312): CodeIgniter\CodeIgniter->run()
#12 C:\xampp2\htdocs\codeigniter\codeigniter4\codeigniter4-framework-9b2cd73\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#13 C:\xampp2\htdocs\codeigniter\codeigniter4\codeigniter4-framework-9b2cd73\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#14 C:\xampp2\htdocs\codeigniter\codeigniter4\codeigniter4-framework-9b2cd73\system\Commands\Server\rewrite.php(49): require_once('C:\\xampp2\\htdoc...')
#15 {main}
CRITICAL - 2024-06-16 14:58:58 --> CodeIgniter\Database\Exceptions\DatabaseException: Unable to connect to the database.
Main connection [MySQLi]: No connection could be made because the target machine actively refused it
[Method: POST, Route: auth/save]
in SYSTEMPATH\Database\BaseConnection.php on line 457.
 1 SYSTEMPATH\Database\BaseConnection.php(604): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1629): CodeIgniter\Database\BaseConnection->query('SELECT 1
FROM `users`
WHERE `email` = :email:
 LIMIT 1', [...], false)
 3 SYSTEMPATH\Validation\StrictRules\Rules.php(228): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\Validation\Validation.php(336): CodeIgniter\Validation\StrictRules\Rules->is_unique('vanessa.cyrilla@strathmore.edu', 'email', [...], null, 'email')
 5 SYSTEMPATH\Validation\Validation.php(201): CodeIgniter\Validation\Validation->processRules('email', 'email', 'vanessa.cyrilla@strathmore.edu', [...], [...], 'email')
 6 SYSTEMPATH\Controller.php(137): CodeIgniter\Validation\Validation->run()
 7 APPPATH\Controllers\Auth.php(72): CodeIgniter\Controller->validate([...])
 8 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Auth->save()
 9 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
10 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 SYSTEMPATH\Boot.php(312): CodeIgniter\CodeIgniter->run()
12 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
13 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
14 SYSTEMPATH\Commands\Server\rewrite.php(49): require_once('C:\\xampp2\\htdocs\\codeigniter\\codeigniter4\\codeigniter4-framework-9b2cd73\\public\\index.php')
INFO - 2024-06-16 14:59:32 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-06-16 14:59:45 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-06-16 15:00:16 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-06-16 15:00:29 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-06-16 15:00:29 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
CRITICAL - 2024-06-16 15:00:42 --> ErrorException: session_start(): open(C:\xampp\tmp\sess_5sej1hnmonum6o31jqouh9ctnf, O_RDWR) failed: No such file or directory (2)
[Method: GET, Route: auth/verifyemail]
in APPPATH\Views\auth\verifyemail.php on line 2.
 1 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'session_start(): open(C:\\xampp\\tmp\\sess_5sej1hnmonum6o31jqouh9ctnf, O_RDWR) failed: No such file or directory (2)', 'C:\\xampp2\\htdocs\\codeigniter\\codeigniter4\\codeigniter4-framework-9b2cd73\\app\\Views\\auth\\verifyemail.php', 2)
 2 APPPATH\Views\auth\verifyemail.php(2): session_start()
 3 SYSTEMPATH\View\View.php(234): include('C:\\xampp2\\htdocs\\codeigniter\\codeigniter4\\codeigniter4-framework-9b2cd73\\app\\Views\\auth\\verifyemail.php')
 4 SYSTEMPATH\View\View.php(237): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 5 SYSTEMPATH\Common.php(1171): CodeIgniter\View\View->render('auth/verifyemail', [], true)
 6 APPPATH\Controllers\Auth.php(25): view('auth/verifyemail')
 7 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Auth->verifyemail()
 8 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
 9 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 SYSTEMPATH\Boot.php(312): CodeIgniter\CodeIgniter->run()
11 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
12 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
13 SYSTEMPATH\Commands\Server\rewrite.php(49): require_once('C:\\xampp2\\htdocs\\codeigniter\\codeigniter4\\codeigniter4-framework-9b2cd73\\public\\index.php')
INFO - 2024-06-16 18:54:11 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-06-16 18:54:11 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
CRITICAL - 2024-06-16 18:55:20 --> ParseError: syntax error, unexpected token "[", expecting identifier or variable or "{" or "$"
[Method: GET, Route: auth/verifyemail]
in APPPATH\Views\auth\verifyemail.php on line 22.
 1 SYSTEMPATH\View\View.php(237): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 2 SYSTEMPATH\Common.php(1171): CodeIgniter\View\View->render('auth/verifyemail', [], true)
 3 APPPATH\Controllers\Auth.php(25): view('auth/verifyemail')
 4 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Auth->verifyemail()
 5 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
 6 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 SYSTEMPATH\Boot.php(312): CodeIgniter\CodeIgniter->run()
 8 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 9 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
10 SYSTEMPATH\Commands\Server\rewrite.php(49): require_once('C:\\xampp2\\htdocs\\codeigniter\\codeigniter4\\codeigniter4-framework-9b2cd73\\public\\index.php')
CRITICAL - 2024-06-16 18:55:38 --> ParseError: syntax error, unexpected single-quoted string "verify_token", expecting identifier or variable or "{" or "$"
[Method: GET, Route: auth/verifyemail]
in APPPATH\Views\auth\verifyemail.php on line 22.
 1 SYSTEMPATH\View\View.php(237): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 2 SYSTEMPATH\Common.php(1171): CodeIgniter\View\View->render('auth/verifyemail', [], true)
 3 APPPATH\Controllers\Auth.php(25): view('auth/verifyemail')
 4 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Auth->verifyemail()
 5 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
 6 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 SYSTEMPATH\Boot.php(312): CodeIgniter\CodeIgniter->run()
 8 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 9 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
10 SYSTEMPATH\Commands\Server\rewrite.php(49): require_once('C:\\xampp2\\htdocs\\codeigniter\\codeigniter4\\codeigniter4-framework-9b2cd73\\public\\index.php')
CRITICAL - 2024-06-16 18:55:58 --> ParseError: syntax error, unexpected token "[", expecting identifier or variable or "{" or "$"
[Method: GET, Route: auth/verifyemail]
in APPPATH\Views\auth\verifyemail.php on line 22.
 1 SYSTEMPATH\View\View.php(237): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 2 SYSTEMPATH\Common.php(1171): CodeIgniter\View\View->render('auth/verifyemail', [], true)
 3 APPPATH\Controllers\Auth.php(25): view('auth/verifyemail')
 4 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Auth->verifyemail()
 5 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
 6 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 SYSTEMPATH\Boot.php(312): CodeIgniter\CodeIgniter->run()
 8 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 9 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
10 SYSTEMPATH\Commands\Server\rewrite.php(49): require_once('C:\\xampp2\\htdocs\\codeigniter\\codeigniter4\\codeigniter4-framework-9b2cd73\\public\\index.php')
